pkg/master/pod_cache.go

```diff
@@ -192,6 +192,7 @@ func (p *PodCache) computePodStatus(pod *api.Pod) (api.PodStatus, error) {
 	} else {
 		newStatus.Info = result.Status.Info
 		newStatus.PodIP = result.Status.PodIP
+		newStatus.Conditions = result.Status.Conditions
```

Comment:
I'm choosing to set the conditions to:
```
[]api.PodCondition{{
	Kind:   api.PodReady,
	Status: api.ConditionUnknown,
}}
```
and remove the pod from load balancing when the node cannot be contacted.